processor/cloningfanoutconnector.go

```diff
@@ -35,65 +28,13 @@ import (
 // clones of data before fanning out, which ensures each consumer gets their
 // own copy of data and is free to modify it.
 
-// CreateMetricsCloningFanOutConnector is a placeholder function for now.
-// It supposed to create an old type connector or a new type connector based on type of provided metrics consumer.
-func CreateMetricsCloningFanOutConnector(mcs []consumer.MetricsConsumerBase) consumer.MetricsConsumerBase {
+// NewMetricsCloningFanOutConnector wraps multiple new type metrics consumers in a single one and clones the data
```

Comment:
Should probably remove the term "new type" which will confuse people since this is the only type now